README.md

```diff
@@ -99,6 +99,8 @@ Martha's `martha_v3` implementation translates requests-to and responses-from th
 
 - ✅ [Jade Data Repo](https://github.com/DataBiosphere/jade-data-repo#readme) (JDR)
     - Prod host: `jade-terra.datarepo-prod.broadinstitute.org`
+    - Alpha host: `data.alpha.envs-terra.bio`
+    - Staging host: `data.staging.envs-terra.bio`
     - Dev host: `jade.datarepo-dev.broadinstitute.org`
```

Comment:
Jade is going to keep adding hosts. We don't have any integration tests hitting these new hosts, nor production-users hitting these new hosts.

So I'd only leave dev-for-us and prod-for-end-users and leave out updating the README for every new Jade host.